jvb/src/main/kotlin/org/jitsi/videobridge/Endpoint.kt

```diff
@@ -488,6 +505,18 @@ class Endpoint @JvmOverloads constructor(
     fun effectiveVideoConstraintsChanged(
         oldEffectiveConstraints: Map<String, VideoConstraints>,
         newEffectiveConstraints: Map<String, VideoConstraints>
+    ) {
+        if (MultiStreamConfig.config.enabled) {
+            effectiveVideoConstraintsChangedV2(oldEffectiveConstraints, newEffectiveConstraints)
```

Comment:
Oh, this is subtle. The java type of the constraints hasn't changed, but the semantics are diffrent (the keys are source names instead of endpoint IDs). Can you add a comment please?

Reply:
yeah, I wish there was a way to declare a type for String and call it a SourceName..

Reply:
actually Kotlin has a typealias feature, I'm curious if that will work: https://kotlinlang.org/docs/type-aliases.html

Reply:
hm the first obstacle is not being able to use an alias from java files

Reply:
I think typealiases are just aliases, they'd be interchangeable. You could extend `String`, but that's probably overkill for something we'll only have temporarily.